### ISSUE TITLE
Daily backup, problem to restore

### ISSUE THREAD

**Issue by c4tom** [dsolonenko/financisto#123]
I had a big problem, I scheduled to backup daily, so I reset my android.

I installed the financisto, configured to access the cloud backups - google drive, and surprise, none of them can restore, even a export in qif format could not restore.

Just a January 2019 backup I got.

what was modified so you can't restore?

Sincerely, thank you in advance if there is a positive response.

**Comment by brunobastosg** [comment]
I have the exact same problem. I got a new phone and couldn't restore my latest Google Drive backup.

The only backup that worked was one from the 5th of February.

Maybe [this commit](https://github.com/dsolonenko/financisto/commit/b59aae523957e3f819033d169693c6d6634aab47) was supposed to fix this problem? Apparently this bug was introduced in version1.8.1.

When will the next version be released?

**Comment by luigui2906** [comment]
Hi,

I was having the same problem. When I restored the backup, the list of accounts was empty and it said I have to select currencies. When I added my currencies, the accounts appeared with all the currencies wrong and mixed. Transaction descriptions were gone. Amounts on the total were displayed incorrectly. I think this is fixed in the 1.8.2 version... what I did was to apply as beta tester on Google play and was able to download the v1.8.2... I restored the same backup and now works correctly. Hope it helps. 

Best.

**Comment by brunobastosg** [comment]
Yeah, can confirm this was fixed in 1.8.2

**Comment by praded2007** [comment]
Yeah, can confirm this was fixed in 1.8.2( not beta), but again does not work in version 1.8.3
